resolwe/flow/models/data.py

```diff
@@ -3,6 +3,7 @@
 import enum
 import json
 import logging
+import os
```

Comment:
Fix the commit message.

Reply:
Now the method was removed and commit message fixed.